src/lib/labels.ts

```diff
@@ -0,0 +1,17 @@
+import { League, Race } from "./api"
+
+export const raceLabels: Record<Race, string> = {
+  infernals: "Infernals",
+  vanguard: "Vanguard",
+  random: "Random",
+}
+
+export const leagueLabels: Record<League, string> = {
```

Comment:
There's similar code in format.ts, which maybe in turn should use this part.

Reply:
Done, I missed it before